source/test/java/com/restfb/types/PostTest.java

```diff
@@ -294,4 +297,30 @@ public void checkV2_8_feedTargetingScheduleUntil() {
     Date ts = examplePost.getFeedTargeting().getRelevantUntilTs();
     assertEquals(1457620560000L, ts.getTime());
   }
+
+  @Test
+  public void checkV2_5_feedTargetingWithRegions() {
+    Post examplePost = createJsonMapper().toJavaObject(jsonFromClasspath("v2_5/post-with-feedtargeting-regions"), Post.class);
```

Comment:
Hopefully I'm able to release the 1.32 until the end of this week, perhaps I manage to release it earlier. I have to check the open issue this evening and afterwards I can decide if there is still something to do. As you can see, there is already the prepare issue.